data/spells/lib/spells.lua

```diff
@@ -348,3 +348,36 @@ function Creature:addPartyCondition(combat, variant, condition, baseMana)
 	return true
 end
 
+function Player:conjureItem(reagentId, conjureId, conjureCount, effect)
+	if not conjureCount and conjureId ~= 0 then
+		local itemType = ItemType(conjureId)
+		if itemType:getId() == 0 then
+			return false
+		end
+
+		local charges = itemType:getCharges()
+		if charges ~= 0 then
+			conjureCount = charges
+		end
+	end
+
+	if reagentId ~= 0 and not self:removeItem(reagentId, 1, -1) then
+		self:sendCancelMessage(RETURNVALUE_YOUNEEDAMAGICITEMTOCASTSPELL)
+		self:getPosition():sendMagicEffect(CONST_ME_POFF)
+		return false
+	end
+
+	local item = self:addItem(conjureId, conjureCount)
+	if not item then
+		self:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
+		self:getPosition():sendMagicEffect(CONST_ME_POFF)
+		return false
+	end
+
+	if item:hasAttribute(ITEM_ATTRIBUTE_DURATION) then
+		item:decay()
+	end
+
+	self:getPosition():sendMagicEffect(effect)
```

Comment:
Suggestion: use `effect or CONST_ME_MAGIC_GREEN` and you will have a default value for effect you can omit on all files. It seems all conjuring has this same effect.